src/clr-addons/clr-addons.module.ts

```diff
@@ -29,6 +29,7 @@ import { ClrTreetableModule } from './treetable/treetable.module';
 import { ClrViewEditSectionModule } from './view-edit-section/view-edit-section.module';
 import { ClrHistoryModule } from './history/history.module';
 import { ClrAutocompleteOffModule } from './autocomplete-off/autocomplete-off.module';
+import { ClrBrandAvatarModule } from './brand-avatar';
```

Comment:
the maximum call stacks error originates from here. Just reference the module file directly and its gone:
import { ClrBrandAvatarModule } from './brand-avatar/brand-avatar.module';